cmd/edge-proxy/main.go

```diff
@@ -131,7 +139,7 @@ func main() {
 	}
 
 	go func() {
-		err := server.StartHTTPTunnel(httpTunnelAddr, externalHTTPProxyURI)
+		err := server.StartHTTPTunnel(httpTunnelAddr, externalHTTPProxyURI, externalHTTPProxyCACert)
```

Comment:
Not a necessary change but it might great to have that - it might be better to wrap these configurations as an object. By doing this, we don't need to change the signature of a function every time when we need to pass additional parameters.

Reply:
Done